start/sw/web-app/laravel5.md

```diff
@@ -9,7 +9,7 @@ order: 4
 
 Laravel is a high-level PHP Web framework. To install Laravel the following needs to be satisfied:
 
-1. PHP >= 5.5.9
```

Comment:
PHP from Fedora is installed. Is it relevant to mention the version? Why not rather remove it altogether?